liquibase-core/src/main/java/liquibase/change/core/AddLookupTableChange.java

```diff
@@ -68,7 +68,7 @@ public void setExistingTableCatalogName(String existingTableCatalogName) {
         this.existingTableCatalogName = existingTableCatalogName;
     }
 
-    @DatabaseChangeProperty(mustEqualExisting ="column.relation.schema")
+    @DatabaseChangeProperty(mustEqualExisting ="column.relation.schema", description = "Name of the database schema of the existing table")
```

Comment:
A more complete description is, "Name of the database schema where the table containing data to extract resides."

Reply:
Yeah, I like more your proposed description. I can make this update if @adrian-velonis1 is happy with it.

Reply:
Go for it @MalloD12!

Reply:
Hey @adrian-velonis1, if you are planning to address some other description updates, would you mind updating this proposed description too? Otherwise, I don't mind making the required changes here.

Thanks,
Daniel.